packages/zowe-explorer/__tests__/__unit__/trees/job/JobActions.unit.test.ts

```diff
@@ -999,7 +999,7 @@ describe("Jobs Actions Unit Tests - Function submitMember", () => {
                     }
                 );
             }
-            expect(mocked(Profiles.getInstance)).toHaveBeenCalledTimes(2 * (o + 1));
+            expect(mocked(Profiles.getInstance)).toHaveBeenCalledTimes(4 * (o + 1) + 2);
```

Comment:
(question) I'm wondering if doing a `mockClear` on `mocked(Profiles.getInstance)` before the for loop would cause the number of calls to satisfy the previously-expected value in the test. Not explicitly requesting changes for this as there may be additional context for why the number of calls has increased.

Reply:
clearing mocks doesn't change anything here